res/TensorFlowLiteRecipes/Net_FullyConnected_Mul_003/test.rule

```diff
@@ -0,0 +1,13 @@
+# This checks if:
+#   Mul(FC(input, weights, _), other)
```

Comment:
there is `bias` in the `test.recipe` file but you've noted as `_`
which I understand this as no bias. This is confusing.

if `003` is with bias, I think we need another `004` without bias.

Reply:
Ah, thanks for the explanation.
`"B"` made the confusion.
It would be nice if you renamed `B` as `mutiplicand` or `scale`.

Anyway, is there existing case WITH BIAS? if not, can you add one?

Reply:
https://github.com/Samsung/ONE/pull/13610 -- here are already merged models, all of them have `fc_bias`.

> It would be nice if you renamed B as mutiplicand or scale.

Will do!